cockroachcloud/managing-cmek.md

```diff
@@ -24,14 +24,16 @@ This section gives a high level overview of the operations involved with impleme
 
 - [Enabling CMEK](#enable-cmek) for a {{ site.data.products.dedicated }} requires several steps:
 
-	- Turning on the {{ site.data.products.dedicated }} CMEK feature.
-	- Provisioning an encryption key with your cloud provider's key management service (KMS).
-	- Granting {{ site.data.products.db }} access to the CMEK i.e., the key you control.
-	- Switching your cluster to use the CMEK.
+  - Turning on the {{ site.data.products.dedicated }} CMEK feature.
+  - Provisioning an encryption key with your cloud provider's key management service (KMS).
+  - Granting {{ site.data.products.db }} access to the CMEK i.e., the key you control.
+  - Switching your cluster to use the CMEK.
```

Comment:
Is there a more precise phrase than "Switching your cluster"? I only have the highest-level understanding from this.